### ISSUE TITLE
Exported data duplication

### ISSUE THREAD

**Issue by ichinaski** [akvo/akvo-flow-mobile#39]
**TL;DR** `Export Data` option does nothing but add duplicate data to the **surveyal** folder

_Every_ survey submission will trigger the sync cycle, meaning a new zip file will be created in the `surveyal` directory in the external storage, containing only the info associated with that surveyInstance. This zip file will be sent over to the server if the internet connection is available.

`Settings > Export Data`, on the other hand, will generate a zip file file containing the surveyInstances not flagged as sent. This means that the resulting zip file **will contain no original data**, since every row in that data.txt will be available through any of the zip files in the `surveyal` folder.

If the users, prior to a bulk upload, manually trigger the export, then zip the **surveyal** folder, the data sent to the server will be the same as just zipping the **surveyal**, without any kind of prior interaction.
